packages/embark-blockchain-connector/src/provider.js

```diff
@@ -1,6 +1,6 @@
 import { __ } from 'embark-i18n';
 const async = require('async');
-const { AccountParser } = require('embark-utils');
+const { AccountParser, dappPath } = require('embark-utils');
```

Comment:
Why do you get dappPath from utils instead of core? Why do we have both actually?

Reply:
That's a bug right there haha.